src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs

```diff
@@ -109,5 +109,10 @@ internal static class Constants
         /// Name used for reference update.
         /// </summary>
         public static string ReferenceUpdateSchemaName = "ReferenceUpdateSchema";
+
+        /// <summary>
+        /// Name used for reference request body
+        /// </summary>
+        public static string ReferenceRequestBodyName = "refBody";
```

Comment:
Should we be worried of potential name conflict with workloads entities in the future? Thoughts on having this as `ODataRef` instead?

Reply:
NVM, workload entities will have namespaces in their names. e.g., `microsoft.graph.user`.